persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/DatabaseExporter.kt

```diff
@@ -19,15 +19,31 @@
 package com.wire.kalium.persistence.backup
 
 import app.cash.sqldelight.db.SqlDriver
+import com.wire.kalium.persistence.DumpContentQueries
+import com.wire.kalium.persistence.dao.UserIDEntity
+import com.wire.kalium.persistence.db.PlatformDatabaseData
+import com.wire.kalium.persistence.db.UserDatabaseBuilder
+import com.wire.kalium.persistence.db.nuke
+import com.wire.kalium.persistence.db.userDatabaseBuilder
+import com.wire.kalium.util.KaliumDispatcherImpl
 
 interface DatabaseExporter {
-    fun beforeBackup()
+    // dump the user DB into a plain DB and return the location of the file
+    fun exportToPlainDB(): String?
+
+    // deleted the plain DB file
+    fun deleteBackupDBFile(): Boolean
```

Comment:
It would be great to have these methods documented to allow autogenerated docs with Dokka